docs/source/api/python/dgl.distributed.rst

```diff
@@ -0,0 +1,76 @@
+.. _api-distributed:
+
+dgl.distributed
+=================================
+
+.. automodule:: dgl.distributed
+
+DGL Distributed
+---------------
+
+Initialization and Exit
+```````````````````````
+
+.. autosummary::
+    :toctree: ../../generated/
+
+    initialize
+    exit_client
```

Comment:
Do we need to expose exit_client to use?
It seems use do not need to care about this.